hikari/commands.py

```diff
@@ -161,8 +161,8 @@ class CommandOption:
     If `builtins.None`, then all channel types will be accepted.
     """
 
-    is_autocomplete: bool = attr.field(default=False, repr=False)
-    """Whether this option is an autocomplete interaction."""
+    autocomplete: bool = attr.field(default=False, repr=False)
+    """Whether this option has autocomplete."""
```

Comment:
Regression here

Reply:
I feel there's a problem with calling this `is_autocomplete` considering the field is saying whether an option is meant to be autocompleted, not that it's the actual autocomplete. 

What about `has_autocomplete` or smth? Pretty inconsistent with the rest of the fields though.

Reply:
Yep, that works even better. We have the style of prefixing booleans with `is_`. `is_autocompleted` will work amazingly

Reply:
Why suddenly `is_autocompleted`?

Reply:
For naming consistency, i think.

Reply:
I feel like making a variable name make sense might be more beneficial than arbitrarily making it consistent with other fields. Surely `has_autocomplete` isn't that far off.